data/hails.txt

```diff
@@ -2045,23 +2047,27 @@ phrase "friendly disabled"
 		
 phrase "friendly disabled"
 	word
-		"Hey! "
+		"Hey!"
+		"You're a friendly, right?"
+		"You answered!"
+		"Good to see a friendly face!"
+		"Hey, please!"
 		""
 		""
-		"You're a friendly, right? "
-		"You answered! "
-		"Good to see a friendly face! "
 		""
-		"Hey, pleaase! "
 		""
 		""
 	word
-		"Our ship is disabled, "
-		"We're kinda stuck here, "
-		"We're in a bit of a bind here, "
-		"We're stuck drifting around in this deathtrap, "
-		"Not to be a bother, but "
-		"Could you come over, and "
+		" "
```

Comment:
the first word has some "" versions, which means such hails "start" at the 3rd word, while these would get this empty space before them
remove this word entirely and add a space after the non-empty versions of word 1